components/validator/custom_validation.rst

```diff
@@ -0,0 +1,5 @@
+.. index::
+    single: Validator; Custom validation
+
+Custom validation

```

Comment:
Nope, Validation is not a mark|country|person and don't refers to the `Validation` class here.